tests/languages/brightscript/directive_feature.test

```diff
@@ -0,0 +1,67 @@
+#const someFlag = true
+#const anotherFlag = false
+#const someOtherFlag = someFlag
+
+#if FeatureA
+#else if FeatureB
+#else
+#end if
+
+#error Your device cannot support this feature.
+
+----------------------------------------------------
+
+[
+	["directive", [
+		["directive", "#const"],
+		["expression", [
+			" someFlag ",
+			["operator", "="],
+			["boolean", "true"]
+		]]
+	]],
```

Comment:
Are these nested directives expected/necessary?

Reply:
I could rename the inner one to `keyword` or do you mean that the wrapping is kinda useless because it contains a BightScript expression?

Reply:
Well, it just seemed odd that a directive is nested inside a directive. It's even weirder further down where the only element inside the directive is another directive. I'm not sure whether that makes sense syntactically for the language, but maybe just dropping the outer wrapping directive would do it?

Reply:
I did this to be consistent with other languages that have compiler directives like C/C++ (even though the tokens are called `macro` there).

Reply:
Ok, that makes some sense, although if the languages aren't really related to each other, then it's not really necessary to make them consistent with each other.

Reply:
It's usually nice to have a wrapper around these semantical units. The wrapper makes it easier to style thinks (some like to have compiler stuff in a different color) and right now, removing the wrapper will change the style because of the `property` alias.

Also, yes. They don't have to be consistent but it's nice that they are.

Reply:
Ok, so we can leave the wrapper if you think that's helpful but maybe the wrapper should be a different name?

Reply:
A different name sounds good. Any suggestions?

btw [this](https://developer.roku.com/es-mx/docs/references/brightscript/language/conditional-compilation.md) is the relevant part of the spec.

Reply:
`directive-expression` or `directive-statement`? Based on the spec, they seem somewhat similar to `if` statements, but for the compiler.

Reply:
I like `directive-statement`. Let's go with that.